### ISSUE TITLE
negative ages/times

### ISSUE THREAD

**Issue by chappjc** [decred/dcrdata#875]
Negative times show up in places (e.g. "-21442s since last block").  This seems to happen when the server is in a different time zone than the client.

**Comment by buck54321** [comment]
Just to add a recent observation, because this bug is maybe weirder than we think. I've seen the block timer in the footer be an hour off on a local server when first firing up dcrdata, only to correct itself when the first new block came in.

**Comment by ZeroASIC** [comment]
This was pretty easy to recreate I just sat there and waited for a new block.
https://alpha.dcrdata.org/address/Dsn86fav4tXZnXF1A6L7ngQyF8NWGbHgXqu#list-view

Notice that the Time UTC column changes to NaN-NaN-NaN 
Here's a screen shot
![screenshot_2018-12-18_16-03-11](https://user-images.githubusercontent.com/4195716/50185946-e507ce00-02de-11e9-9c52-ea46c1a0075f.png)

**Comment by ZeroASIC** [comment]
Could this be related to issue #867?  That issue has a screenshot with NaN-NaN-NaN too.

**Comment by chappjc** [comment]
Yes, that's the time issue in https://github.com/decred/dcrdata/issues/876.  It's not clear it is related to this issue, but it's possible.  I suspect Batman is to blame for that one.

**Comment by ZeroASIC** [comment]
I just double checked and the Time Since Last Block was correct on page load, but went negative after about 30 seconds.  Time UTC was still correct this time.

**Comment by chappjc** [comment]
A little debugging of alpha.dcrdata.org in FF:

![image](https://user-images.githubusercontent.com/9373513/50195832-5b6bf680-0306-11e9-97d6-e70ce0b7f250.png)

The client machine's time is about `1545187586`, whereas the `unixTime` fed to `timeSince` should be to the UTC block times (e.g. for block 301975, `"time": 1545187356`).

The input to `timeSince` is clearly incorrect:

`humanize.timeSince(Date.parse(el.dataset.age) / 1000)` in `setAges`.  The value of `el.dataset.age` in that is `"2018-12-19 02:42:36"`, which does not indicate time zone.

`Date.parse` needs to be given time zone information, or it should assume UTC, not local time.
